.github/workflows/protocol-tests.yaml

```diff
@@ -48,9 +48,10 @@ jobs:
 
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1.0.9
-
-      # weird bug with forge needing to be built twice
-      - run: forge install
```

Comment:
In my latests tests on gh actions, I didn't think this is required, see latest one https://github.com/TempleDAO/temple/actions/runs/5735218316